src/plugin.ts

```diff
@@ -22,7 +23,17 @@ export async function startStopTask(inputs: PluginInputs, env: Env) {
   context.adapters = createAdapters(supabase, context);
 
   if (context.eventName === "issue_comment.created") {
-    await userStartStop(context);
+    try {
+      return await userStartStop(context);
+    } catch (err) {
+      let errorMessage;
+      if (err instanceof LogReturn) {
+        errorMessage = context.logger.error(`Failed to run comment evaluation. ${err.logMessage?.raw || err}`, { err });
+      } else {
+        errorMessage = context.logger.error(`Failed to run comment evaluation. ${err}`, { err });
+      }
+      await addCommentToIssue(context, `${errorMessage?.logMessage.diff}\n<!--\n${JSON.stringify(errorMessage?.metadata, null, 2)}\n-->`);
```

Comment:
It is a bit risky to stringify the object without making sure that the content is not escaped. If there is any special character (`<`, `>`, `-->` etc) there is a change the whole comment break. It would be great to sanitize it before.